dbt_project.yml

```diff
@@ -102,6 +102,10 @@ models:
       ethereum:
         +schema: balances_ethereum
         +materialized: view
+  
+    labels:
+      +schema: labels
+      +materialized: view
```

Comment:
if you leave out the subdirectories (i.e. ethereum) here, will all schemas in the 'labels' directory default to 'labels'?